NuGet.Config

```diff
@@ -21,8 +22,11 @@
       <package pattern="messagepack" />
       <package pattern="messagepack.annotations" />
       <package pattern="messagepackanalyzer" />
-      <package pattern="microsoft.*" />
-      <package pattern="Microsoft.VisualStudio.*" />
+      <package pattern="Microsoft.*" />
+      <package pattern="Microsoft.NETCore.App.Host.linux-x64" />
+      <package pattern="Microsoft.NETCore.App.Host.osx-x64" />
+      <package pattern="Microsoft.NETFramework.ReferenceAssemblies.net45" />
+      <package pattern="Microsoft.NETFramework.ReferenceAssemblies.net461" />
```

Comment:
All the package specific patterns would already be matched by microsoft.* if they're not mapped by another source, and it doesn't make sense to me for any Microsoft.NetCore.* or Microsoft.NETFramework.* pattern to be mapped to any other source.